src/platform/nrfconnect/KMUKeyAllocator.h

```diff
@@ -0,0 +1,136 @@
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+#include <cracen_psa_kmu.h>
+#include <crypto/PSAKeyAllocator.h>
+// Define the number of slots per NOC and ICD key.
```

Comment:
nit: but can we please make some more new lines in between of blocks for better readability